docs/content/reference/concepts-and-personas.md

```diff
@@ -0,0 +1,45 @@
+# Concepts and Personas
+
+## Concepts
+
+**Management Cluster**
+
+An OpenShift cluster which hosts HyperShift and zero to many Hosted Clusters.
+
+**Hosted Cluster**
+
+An OpenShift API endpoint managed by HyperShift.
+
+**Hosted Control Plane**
+
+An OpenShift control plane running on the Management Cluster which is exposed by a Hosted Cluster's API endpoint. The component parts of a control plane include etcd, apiserver, kube-controller-manager, vpn, etc.
+
+**Hosted Control Plane Infrastructure**
+
+Resources on the Management Cluster or external cloud provider which are prerequisites to running Hosted Control Plane processes.
+
+**Management Cluster Infrastructure**
+
+Network, compute, storage, etc. of the Management Cluster.
+
+**Hosted Cluster Infrastructure**
+
+Network, compute, storage, etc. that exist in customer cloud account.
```

Comment:
By these two definitions the kube-apiserver LB would be part of the Management Cluster Infrastructure, since it is not in the customers cloud account which might be confusing or WDYT?

Reply:
Actually wouldn't it fall under `Hosted Control Plane Infrastructure`?

mm not really sure how make these three more specific.